src/client/src/orangehrmClaimPlugin/pages/ClaimEvent.vue

```diff
@@ -0,0 +1,251 @@
+              type="select"
+              :label="$t('general.status')"
+              :options="ClaimEventStatuses"
+            />
+          </oxd-grid-item>
+        </oxd-grid>
+      </oxd-form-row>
+      <oxd-divider />
+      <oxd-form-actions>
+        <oxd-button
+          display-type="ghost"
+          :label="$t('general.reset')"
+          @click="onClickReset"
+        />
+        <oxd-button
+          class="orangehrm-left-space"
+          display-type="secondary"
+          :label="$t('general.search')"
+          type="submit"
+        />
+      </oxd-form-actions>
+    </oxd-form>
+  </oxd-table-filter>
+  <br />
+  <div class="orangehrm-paper-container">
+    <div class="orangehrm-header-container">
+      <oxd-button
+        :label="$t('general.add')"
+        icon-name="plus"
+        display-type="secondary"
+        @click="onClickAdd"
+      />
+    </div>
+    <table-header
+      :total="total"
+      :loading="isLoading"
+      :selected="checkedItems.length"
+    />
+    <div class="orangehrm-container">
+      <oxd-card-table
+        v-model:selected="checkedItems"
+        v-model:order="sortDefinition"
+        :items="items.data"
+        :headers="headers"
+        :selectable="true"
+        :clickable="false"
+        :loading="isLoading"
+        row-decorator="oxd-table-decorator-card"
+      />
+    </div>
+    <div class="orangehrm-bottom-container">
+      <oxd-pagination
+        v-if="showPaginator"
+        v-model:current="currentPage"
+        :length="pages"
+      />
+    </div>
+  </div>
+  <delete-confirmation ref="deleteDialog"></delete-confirmation>
+</template>
+
+<script>
+import {ref, computed} from 'vue';
+import {APIService} from '@/core/util/services/api.service';
+import usePaginate from '@ohrm/core/util/composable/usePaginate';
+import useSort from '@ohrm/core/util/composable/useSort';
+import {navigate} from '@/core/util/helper/navigation';
+import DeleteConfirmationDialog from '@ohrm/components/dialogs/DeleteConfirmationDialog.vue';
+
+const defaultFilters = {
+  name: '',
+  status: null,
+};
+
+const defaultSortOrder = {
+  'claimEvent.name': 'ASC',
+  'claimEvent.status': 'DESC',
+};
+
+export default {
+  components: {
+    'delete-confirmation': DeleteConfirmationDialog,
+  },
+  setup() {
+    const filters = ref({...defaultFilters});
+
+    const {sortDefinition, sortField, sortOrder, onSort} = useSort({
+      sortDefinition: defaultSortOrder,
+    });
+
+    const serializedFilters = computed(() => {
+      return {
+        name: filters.value.name,
+        status: filters.value.status ? filters.value.status?.id === 1 : null,
+        sortField: sortField.value,
+        sortOrder: sortOrder.value,
+      };
+    });
+
+    const claimEventDataNormalizer = (data) => {
+      return data.map((item) => {
+        return {
+          name: item.name,
+          description: item.name,
+          status: item.status ? 'Active' : 'Inactive',
```

Comment:
Active/Inactive should be translated. our i18n helper `$t` is not available inside the `setup()`. there for you must use [usei18n](https://github.com/orangehrm/orangehrm/blob/main/src/client/src/core/util/composable/usei18n.ts) composable for this.

Check this reference. 
https://github.com/orangehrm/orangehrm/blob/main/src/client/src/orangehrmRecruitmentPlugin/pages/ViewJobVacancy.vue#L172